app/collections/settings.jsx

```diff
@@ -84,6 +89,37 @@ export default class CollectionSettings extends React.Component {
       });
   }
 
+  handleDescriptionInputChange() {
```

Comment:
Since the AutoSave component is handling the resource update and save, then a lot of what this method is doing isn't necessary. Tracking the description length and communicating that error is helpful because all the AutoSave component does on error is turn the background red around the input and doesn't actually say what's wrong.